README.es-ES.md

```diff
@@ -0,0 +1,567 @@
+
+**Herramientas**
+
+- [Webpack](https://github.com/webpack/webpack) - permite la carga de módulos de forma asíncrona.
+- [ngx-quicklink](https://github.com/mgechev/ngx-quicklink) - router preloading strategy which automatically downloads the lazy-loaded modules associated with all the visible links on the screen
+- [ngx-quicklink](https://github.com/mgechev/ngx-quicklink) - Estrategía de precarga basada en el enrutado la cual descarga de forma diferida (lazy load) los módulos asociados a los links visibles en pantalla.
+
+### No cargar de forma diferida la ruta por defecto
+
+Supongamos que tenemos la siguiente configuración de enrutado:
+
+```ts
+// Bad practice
+const routes: Routes = [
+  { path: '', redirectTo: '/dashboard', pathMatch: 'full' },
+  { path: 'dashboard',  loadChildren: './dashboard.module#DashboardModule' },
+  { path: 'heroes', loadChildren: './heroes.module#HeroesModule' }
+];
+```
+
+La primera vez que el usuario abre la aplicación usando la url: https://example.com/ será redirigido hacia `/dashboard` la cual disparará la carga diferida (lazy-route) con la ruta `dashboard`. Para que Angular renderice el componente del módulo, deberá descargar el fichero `dashboard.module` y todas sus dependencias. Después, el archivo será parseado por la JavaScript VM y será evaluado.
+
+Disparar solicitudes HTTP extra y ejecutar cálculos innecesarios durante la carga inicial es una mala práctica y ralentizará el renderizado de la página inicial. Considere declarar la ruta de la página predeterminada como no diferida (non-lazy).
+
+### Caché
+
+El almacenamiento en caché es otra práctica común que pretende acelerar nuestra aplicación aprovechando la heurística de que si se solicitó un recurso recientemente, podría solicitarse nuevamente en un futuro próximo.
+
+Para almacenar datos en caché normalmente usamos un mecanismo de almacenamiento en caché personalizado. Para el almacenamiento en caché de assets, podemos utilizar el almacenamiento en caché estándar del navegador o el uso de Service Workers con el [API CacheStorage](https://developer.mozilla.org/en-US/docs/Web/API/Cache).
+
+### Shell de la Aplicación
+
+Para hacer más rápido el rendimiento percibido de su aplicación, usar una [Application Shell](https://developers.google.com/web/updates/2015/11/app-shell).
+
+El shell de la aplicación es la interfaz de usuario mínima que mostramos a los usuarios para indicarles que la aplicación se entregará pronto. Para la generación dinámica del shell de la aplicación puedes utilizar Angular Universal con directivas personalizadas que de forma condicional mostrarán elementos dependiendo de la plataforma de renderizado (por ejemplo, ocultar todo excepto el shell cuando usemos `patform-server`).
```

Comment:
typo --> 'platform-server'

Reply:
I do not understand what should change here, could you explain it a little more please? Thank you

Reply:
> ... usemos `patform-server`

debe cambiarse por 

usemos `platform-server`

(notar la L en pLatform) ;)

Reply:
OMG! thanks!